native/docs/troubleshooting.md

```diff
@@ -127,3 +127,22 @@ The following error occurs:
 
 The app is installed correctly on the android emulator but crashes without any error message on start up.
 To solve this issue delete /android/app/build and android/.gradle folder.
+
+## XCode can't find 'your/directory/native/ios/Integreat/de.lproj/InfoPlist.strings'
+
+These translations should usually be generated during the build, if they aren't, generate them manually with
+`yarn workspace translations manage write-plist Integreat --translations "../translations/translations.json" --destination ../native/ios/Integreat`
+
+Then run `bundle exec pod install` in /ios, clean the Build folder in XCode(Shift + Cmd + K), restart XCode, and try again.
+
+## XCode can't find $BUILD_CONFIG_APP_ICON
+
+Should also be generated during the build, manual command here: `yarn workspace build-configs manage write-xcconfig integreat ios --directory ../native/ios`
+
+Then run `bundle exec pod install` in /ios, clean the Build folder in XCode(Shift + Cmd + K), restart XCode, and try again.
+
+
+## Other assorted XCode problems
+
+It often helps to rerun `yarn install` in /native and `bundle exec pod install` in /ios, then clean the build
+folder in XCode (Shift + Cmd + K), and then restart XCode.
```

Comment:
thx for adding hints to the troubleshooting. Could you please  add another small part

## M1 chip problems
- some pods causing problems with arm chips and remove the excluded architecture while `pod install`, therefore you have to ensure that under "Build Settings" -> "Architecture" the "Excluded architectures are added for Debug & Release:
`Any iOS Simulator SDK: arm64`